### ISSUE TITLE
Cart adds shipping subtotal to cart subtotal

### ISSUE THREAD

**Issue by yhortuk** [lunarphp/lunar#829]
- Lunar version: v0.2-rc1

### Expected Behaviour:

Cart subtotal shouldn't include shipping costs (at least it is what the field description states).

### Actual Behaviour:

`$cart->subTotal` returns products subtotal including the shipping costs.

### Steps To Reproduce:

1) Create a cart instance
2) Add non zero shipping option to the cart
3) Call `$cart->calculate()`
4) Call `$cart->subTotal`

**Comment by glennjacobs** [comment]
Yes, I agree with this. It seems the accepted standard is to have the sub-total before shipping is applied. 

I'm going to change to enhancement, as it's not technically a bug.

**Comment by ryanmitchell** [comment]
@glennjacobs This is biting us too. Is this something youre working on or do you need a PR for it?

**Comment by glennjacobs** [comment]
> @glennjacobs This is biting us too. Is this something youre working on or do you need a PR for it?

I would be looking at this for the 0.3 release.

**Comment by ryanmitchell** [comment]
I've opened a PR anyway in case you want to sneak it into 0.2
😄
